lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,239 @@
+/*css idea source: https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/animate-elements-at-variable-rates*/
+@import "color.variables";
+@import "size.variables";
+@import "font.variables";
+@import "settings.responsive";
+@import "flex.responsive";
+
+body {
+  background-color: $black;
+  width: $hundredper;
+  height: $hundredper;
+  position: fixed;
+  margin: 0;
+  top: 0;
+  left: 0;
+  font-family: $redhat, $sans;
+  @include media-query("mobile") {
+    width: auto;
+    height: $hundredper;
+  }
+
+  .main {
+    /*twinkling stars style*/
+    &__stars {
+      height: $seven;
+      width: $seven;
+      border-radius: $hundredper;
+      animation-iteration-count: infinite;
+    }
+    /*shooting stars style*/
+    &__shootingstar {
+      height: $ten;
+      width: $ten;
+      border-radius: $hundredper;
+      animation-iteration-count: infinite;
+    }
+    /*shooting stars animation*/
+    &__starshooter1 {
+      margin-top: $zeroper;
+      margin-left: $zeroper;
+      animation-name: shootingstar;
+      animation-duration: 3s;
+    }
+
+    &__starshooter2 {
+      margin-top: $zeroper;
+      margin-left: $zeroper;
+      animation-name: shootingstar;
+      animation-duration: 1s;
+    }
+
+    &__starshooter3 {
+      margin-top: $zeroper;
+      margin-left: $zeroper;
+      animation-name: shootingstar;
+      animation-duration: 4s;
+    }
+    /*twinkling stars animation*/
+    &__star-0 {
+      margin-top: $threeper;
```

Comment:
Love all your star animations, seriously so clever & cool! Very fitting of your site